httpstats/transport.go

```diff
@@ -7,6 +7,8 @@ import (
 	"github.com/segmentio/stats"
 )
 
+const contextKeyReqTags = "segmentio_httpstats_req_tags"
```

Comment:
How about replacing this with an internal empty type to avoid any risk of conflicts? For example
```go
type tagsKey struct{}
```